osm2streets/src/intersection.rs

```diff
@@ -196,10 +196,15 @@ impl StreetNetwork {
         intersection.roads = road_centers.into_iter().map(|(r, _, _)| r).collect();
     }
 
-    /// Updates the derived properties of an intersection.
-    ///
+    /// Updates an intersection's derived properties -- geometry (and attached roads) and
```

Comment:
I think we always want to do these two when something attached to the intersection changes. We only need to do `sort_roads` in a few cases like inserting a new road though. Maybe in #159, we just start granularly tracking a dirty set of intersections and call this?